### ISSUE TITLE
Send to on-chain wallet

### ISSUE THREAD

**Issue by iBobik** [dennisreimann/btcpayserver#13]
Some users asking me to forward their LN wallets to their HW wallet in some conditions (weekly or on some amount). Would be nice if there will be action (in UI and in API) for [Reverse submarine swap](https://docs.boltz.exchange/en/latest/lifecycle/#reverse-submarine-swaps) moving their LN bank money to XPUB configured in on-chain wallet of their store.

**Comment by ishristov** [comment]
Oh wow, if that's possible, it would be super useful.

**Comment by Kukks** [comment]
I think a better flow would be to allow "withdrawals" through pull payments, which already support multiple payment method options (LN, LNURL, LN Address, BTC, Payjoin, etc)

**Comment by Kukks** [comment]
> I think a better flow would be to allow "withdrawals" through pull payments, which already support multiple payment method options (LN, LNURL, LN Address, BTC, Payjoin, etc)

How can this work? 
Server admin needs to create a store, configure store and then also have a way to tell LNBank that this store is the administration store.  Pull payment would be created on this store, and the amount is deducted from the balance straight away (and the pull payment is pre-approved). 

This means we can even generate invoices on the store to "top up" your lnbank balance.

**Comment by iBobik** [comment]
Pull payments can send by sumbarine swap?

Honza Pobořil

**Comment by Kukks** [comment]
No, there is no submarine swap involved. Submarine swaps are not really
useful here imo since this is a custodial solution which has trust built
into the model.
